src/script/components/Conversation/ConversationCells/useRefreshCellsState/useRefreshCellsState.ts

```diff
@@ -50,9 +50,9 @@ export const useRefreshCellsState = ({
     fetchCountRef.current += 1;
   }, [conversationRepository, conversationQualifiedId]);
 
-  useEffect(() => {
+  useCallback(async () => {
     if (isInitialMount.current) {
-      void refreshCellsState();
+      await refreshCellsState();
```

Comment:
calling an async function in the `useEffect` here creates the problem no matter what `reffreshCellState()` does, it will create the display issue.